src/index.ts

```diff
@@ -13,19 +13,41 @@ const dir = __dirname.endsWith('dist/src') ? '../' : '';
 const schemaFile = path.join(__dirname, `${dir}../src/schema.gql`);
 const schema = fs.readFileSync(schemaFile, 'utf8');
 
-const checkpoint = new Checkpoint(config, writer, schema, checkpoints, {
+const checkpoint = new Checkpoint(config, writer, schema, {
   logLevel: LogLevel.Info,
   prettifyLogs: process.env.NODE_ENV !== 'production'
 });
 checkpoint.reset();
```

Comment:
After this PR get's merged, we should be able to safely remove this call to `reset()` and instead seed checkpoints (or metadata) in the database if we need to speed things up.

Reply:
But you still need to delete the tables, we could keep the `reset()` function but just delete the generated tables and not the `_checkpoints` table